scripts/ae_integration_test.sh

```diff
@@ -25,10 +25,20 @@ deployDir=$testAppDir/target/deploy
 
 imageUnderTest=$1
 if [ -z "${imageUnderTest}" ]; then
-  echo "Usage: ${0} <image_under_test>"
+  echo "Usage: ${0} <image_under_test> [gae_deployment_version]"
   exit 1
 fi
 
+# for local tests it makes sense sometimes to pin the deployment to an
+# active version as that will speed up the deployment, for CI/CD this feature
+# is not recommended
+gaeDeploymentVersion=$2
+if [ ! -z "${gaeDeploymentVersion}" ]; then
```

Comment:
I believe `[ -n $foo ]` or just `[ $foo ]` are equivalent to `[ ! -z $foo ]`, and might be a bit cleaner.